app/models/issue.rb

```diff
@@ -39,7 +39,8 @@ class Issue < ApplicationRecord
   scope :ordered, -> { order(created_at: :desc) }
   scope :approved, -> { where(status: :opened) }
   scope :closed, -> { where(status: :closed) }
-  scope :find_issues, ->(a) { where('title like ?', "%#{a}%") }
+  search_query = 'title like :args OR users.last_name like :args'
```

Comment:
do you need this variable `search_query ` in other places?
if not just put it dirrectly in find_issues scope